src/tests/ftest/server/metadata.py

```diff
@@ -236,24 +246,21 @@ def run_dummy_metadata_workload(self, duration=150):
 
         return True
 
-    def test_metadata_fillup(self):
-        """JIRA ID: DAOS-1512.
-
-        Test Description:
-            Test to verify no IO happens after metadata is full.
+    def metadata_fillup(self, svc_ops_enabled=True):
+        """Run test to verify no IO happens after metadata is full.
 
-        Use Cases:
-            ?
+        Args:
+            svc_ops_enabled (bool): Pool create properties svc_ops_enabled, default to True.
 
-        :avocado: tags=all,full_regression
-        :avocado: tags=hw,large
-        :avocado: tags=server,metadata
-        :avocado: tags=ObjectMetadata,test_metadata_fillup
         """
-        self.create_pool()
-        svc_ops_entry_age = self.pool.get_property("svc_ops_entry_age")
-        if not self.run_dummy_metadata_workload(duration=svc_ops_entry_age):
-            self.fail("failed to run dummy metadata workload")
+        self.log_step("1. Create pool with properties svc_ops_enabled: {}".format(svc_ops_enabled))
+        self.create_pool(svc_ops_enabled=svc_ops_enabled)
+        # Run dummy_metadata_workload when feature is enabled
+        if svc_ops_enabled:
+            self.log.info("svc_ops_enabled enabled, run dummy_metadata_workload")
+            svc_ops_entry_age = self.pool.get_property("svc_ops_entry_age")
+            if not self.run_dummy_metadata_workload(duration=svc_ops_entry_age):
+                self.fail("#1. failed to run dummy metadata workload")
```

Comment:
Similar - we shouldn't add numbers to `self.fail` because they probably won't correspond to the numbers that `log_step` automatically adds

Reply:
Same.